src/components/StartCheckoutButton.tsx

```diff
@@ -86,20 +86,29 @@ export default function StartCheckoutButton() {
           </div>
         </div>
       )}
-      <Link
-        to={claims != null ? '/checkout/details' : '/cart/checkout'}
-        tabIndex={-1}
-        onClick={() => {
-          if (claims != null) {
+      {claims !== null && (
+        <Button
+          variant="primary"
+          disabled={isCartEmpty}
+          onClick={() => {
             clearCart();
-          }
-        }}
-      >
-        <Button variant="primary" disabled={isCartEmpty}>
+            logout({
+              redirectUri: window.location.origin + '/checkout/details',
```

Comment:
`/checkout/details` has to be registered as a callback URL, @mickhansen can help with that :)

Reply:
Instead of registering a bunch of redirect_uris it's also possible to use `state` to handle where the user should return to.

Reply:
Fixed in https://github.com/criipto/demo-cool-delivery-react/pull/5/commits/4becadf8843b5d1f146c14a279428010f2447719 and https://github.com/criipto/demo-cool-delivery-react/pull/5/commits/918b681e927866eb1e759fff9136304d14a2d837